baseapp/baseapp.go

```diff
@@ -220,6 +241,29 @@ func (app *BaseApp) setDeliverState(header abci.Header) {
 	}
 }
 
+// setConsensusParams memoizes the consensus params.
```

Comment:
I think this further alludes to the statement above. I don't see the direct relationship between these params and the ones used via the context. I think at the very least the godoc should allude to what these consensus params are for.

Reply:
Good question.  The types/Context consensus params were never used.  We don't need them in the context for now, and even if we need them we'd want to modify them via a keeper, so for now will delete types/Context.ConsensusParams.